internal/distro/rhel8/imagetype.go

```diff
@@ -431,6 +431,12 @@ func (t *imageType) checkOptions(customizations *blueprint.Customizations, optio
 			if err := customizations.CheckAllowed(allowed...); err != nil {
 				return warnings, fmt.Errorf("unsupported blueprint customizations found for boot ISO image type %q: (allowed: %s)", t.name, strings.Join(allowed, ", "))
 			}
+		} else if t.name == "edge-raw-image" {
+			allowed := []string{"User", "Group"}
+			if err := customizations.CheckAllowed(allowed...); err != nil {
+				return warnings, fmt.Errorf("unsupported blueprint customizations found for image type %q: (allowed: %s)", t.name, strings.Join(allowed, ", "))
+			}
+			// TODO: consider additional checks, such as those in "edge-simplified-installer"
```

Comment:
This check is run inside the block for the condition `t.bootISO && t.rpmOstree` but `edge-raw-image` isn't a boot ISO.

Reply:
good catch!